### DISCUSSION TITLE
Updated

### DISCUSSION THREAD

**Discussion by tommiefr** [nicholas-opensource/OpenWrt-Autobuild#12] [General]
Is this release still being updated?
https://github.com/nicholas-opensource/OpenWrt-Autobuild/releases

**Comment by nicholas-opensource** [comment]
It will be released in few days.
rc2 now is coming

**Comment by tommiefr** [comment]
Sorry for the question but is this the new built:

https://github.com/nicholas-opensource/OpenWrt-Autobuild/actions/runs/892596506

**Reply by nicholas-opensource** [reply]
This workflow is the test build, and now I am testing this version. So far everything is working fine~
If you need the latest firmware, you can download it in action workflow. Of course, they are all untested.

**Comment by tommiefr** [comment]
Ok great! Than I'll wait for the final version here.
This built will be special and specific for the nanopi r2s?
What specific has been changed over the original openwrt? And important, now I use snapshot builds and I need to re-configure everytime I update, can I simply update these builds?

**Reply by nicholas-opensource** [reply]
The main change is to fix the RAM frequency of R2S.It will lock at 333MHz under the original openwrt. For other specific changes, please refer to readme and release note, or you can check the steps of scripts/01 02

You can save the tar backup to restore the system, which is recommended. In addition, you can also use sysupgrade to keep config files

**Comment by tommiefr** [comment]
Great! Thanks for the update! Can you tell me the exact difference between the 2 versions of the r2s version? 
Can't find exactly what it is...

**Reply by nicholas-opensource** [reply]
Bingbing is made for China mainland users, contains a special Chinese software and replace packages update source to Chinese mirror site. Hathaway is the general version, no extra services included
